README.md

```diff
@@ -21,15 +21,17 @@ See also [this issue](https://github.com/treasure-data/omnibus-td-agent/issues/2
   * Any host OS that is supported by Docker
     * Debian buster or Ubuntu 18.04 are recommended
   * [Docker](https://docs.docker.com/install/)
-  * Ruby 2.4 or later
+  * Ruby 2.5 or later
+  * Bundler 2.2.0.rc1 (Install by `gem install bundler -v 2.2.0.rc.1`)
```

Comment:
Oops, I already merged it, it should be 2.2.0.rc1 **or later**?

Reply:
Ah, it will be better to mention "or later". I'll fix it.